darwin/importer/formats/nifti.py

```diff
@@ -529,9 +529,14 @@ def process_nifti(
         pixdims: tuple of nifti header zoom values.
     """
     img = correct_nifti_header_if_necessary(input_data)
-    img = nib.funcs.as_closest_canonical(img)
-    data_array = nib.orientations.apply_orientation(img.get_fdata(), ornt)
-    pixdims = img.header.get_zooms()
+    orig_ax_codes = nib.orientations.aff2axcodes(img.affine)
+    orig_ornt = nib.orientations.axcodes2ornt(orig_ax_codes)
+    transform = nib.orientations.ornt_transform(orig_ornt, ornt)
+    reoriented_img = img.as_reoriented(transform)
+
+    data_array = reoriented_img.get_fdata()
+    pixdims = reoriented_img.header.get_zooms()
+
```

Comment:
These changes to `process_nifti` are unrelated and make the function more explicitly able to handle arbitrarily oriented outputs, even though this is not needed for importing to LPI. 

Before, the pixdims were taken from volume reoriented to RAS, meaning the output orientation needed to be in RAS order for the pixdims to be ordered correctly. Now the output's axes can be any order.